incubator/openvpn/values.yaml

```diff
@@ -48,6 +48,8 @@ persistence:
   size: 1Gi
 
 easyrsa:
+  clientCertExpire: "3650"
+  serverCertExpire: "3650"
```

Comment:
I think it's confusing that there's both `EASYRSA_CA_EXPIRE` and `serverCertExpire`. 

What if we default `EASYRSA_CERT_EXPIRE= EASYRSA_CA_EXPIRE` when `OVPN_CALLER=init`